### ISSUE TITLE
Show recent contributors on the comcast.github.io home page

### ISSUE THREAD

**Issue by shillasaebi** [Comcast/Comcast.github.io-archive#283]
Can we show contributors to the Comcast org and 2 other orgs that are Comcast open source projects listed on our home page, similar to our Hacktoberfest page? https://comcast.github.io/hacktoberfest) Maybe limit up to 20 profiles to load so it doesn't take up too much space?

Orgs to pull recent contributor data from: 
https://github.com/xmidt-org
https://github.com/vinyldns
https://github.com/Comcast

Thank you!

**Comment by avidudi** [comment]
Hi @shillasaebi, there's already a section showing contributors on the home page. So, only recent contributors need to appear?

**Comment by shillasaebi** [comment]
We don't actually have a section on comcast.github.io that shows the recent contributors. We only have it on the Hacktoberfest page as of now. Thank you for looking into this for us!

**Comment by avidudi** [comment]
Yes, I see contributors on Hacktoberfest page. I can take this issue.

**Comment by vaish1707** [comment]
I would also like to contribute to this issue.

**Comment by sureshmangs** [comment]
Can we create the recent contributor's section at the bottom of the homepage?

**Comment by JohnRiv** [comment]
> Can we create the recent contributor's section at the bottom of the homepage?

Yes, between the "Overall Stats" and "Comcast OS Projects" sections
